crdt/ctree.go

```diff
@@ -870,6 +871,17 @@ func (t *CausalTree) SetCursor(i int) error {
 	return nil
 }
 
+// +--------------------------------------+
+// + Operations - Atom Priority constants |
+// +--------------------------------------+
+const (
+	InsertCharPriority    = 0
```

Comment:
These constants don't need to be exported, use minuscule for the first letter

Reply:
Done.